defaults/defaults.go

```diff
@@ -29,4 +29,12 @@ const (
 	// DefaultSnapshotterNSLabel defines the namespace label to check for the
 	// default snapshotter
 	DefaultSnapshotterNSLabel = "containerd.io/defaults/snapshotter"
+	// DefaultWindowSize defines the default window size for stream
```

Comment:
Would you mind to provide the reproduce step on this?
The window size is used to control traffic and gRPC-go has auto-tune based on RTT.

REF: https://github.com/grpc/grpc-go/blob/master/internal/transport/bdp_estimator.go

Reply:
And it might be related to https://github.com/grpc/grpc-go/issues/670. 
But I would like to see how to reproduce it.

Reply:
oh sorry, I don't have clear steps to reproduce it, I believe the original intent of this PR came from https://github.com/earthly/earthly/issues/1016 , This should be the same issue. please see https://github.com/containerd/containerd/issues/6190 and https://github.com/earthly/buildkit/pull/42

Reply:
I didn't see how to reproduce it. So I think we can understand the behaviour before merge.
It looks like it is related to gRPC http2 stream which doesn't update the windows in time.
I am not a fan to add a unknown value in here.

Reply:
Or provides option to client to update the window size instead of fixed number here.